test/integration-test.py

```diff
@@ -58,7 +58,7 @@ def expect(target_time, num, table):
         tool = "expired-authz-purger2"
         out = get_future_output([
             "./bin/expired-authz-purger2", "--single-run",
-            "--config", "cmd/expired-authz-purger2/config.json"], target_time)
+            "--config", f"{config_dir}/expired-authz-purger2.json"], target_time)
```

Comment:
This formatting directive is new to me! Seems neat, but let's stick to local style, and then do a pass through to update to the fancier new style if we want it.

Reply:
Fair enough! I just reflexively reach for F-strings.